### DISCUSSION TITLE
Is the brightness of the initial selection screen configurable?

### DISCUSSION THREAD

**Discussion by mrdsaunders** [flameshot-org/flameshot#3175] [Q&A] [answered]
After instigating 'Take screenshot', the darkened screen appears (which indicates selection mode), and upon selecting an area, the selected area has the same brightness as the existing screen.

To my eyes/brain/preference, the darkened screen is too dark and I have to squint/strain to see the area I am selecting. I cannot see anything in configuration.

This is my first week using Linux and open source and I absolutely love the app. Is this the correct channel to ask this question, and do such things as feature requests exist, and do I post them here?

**Comment by mmahmoudian** [comment, marked as answer]
> This is my first week using Linux and open source

Welcome to the beautiful world of Linux and FLOSS

> I absolutely love the app.

We are glad to hear that. We hope you enjoy it and it improve your productivity

> Is this the correct channel to ask this question

You can ask questions here, or [as an issue](https://github.com/flameshot-org/flameshot/issues/new) (not preferred, but it is common practice on Github). Alternatively you can use Fediverse (e.g [Mastodon](https://fosstodon.org/@flameshot), [Matrix](https://matrix.to/#/#flameshot-org:matrix.org)) to ask us questions.

> do such things as feature requests exist, and do I post them here?

You can request features by opening a feature request issue from the [issue chooser](https://github.com/flameshot-org/flameshot/issues/new/choose). But please before posting a new issue, make sure you have searched among open and closed issues and discussions to avoid creating a duplicate.

> I cannot see anything in configuration.

I hope this helps:

![image](https://user-images.githubusercontent.com/390889/233648934-3db1d789-0de5-499a-9cb2-63cf00b61ead.png)

Alternatively you case set the value manually in the `~/.config/flameshot/flameshot.ini`:

![image](https://user-images.githubusercontent.com/390889/233651325-4db91efc-c847-4888-af15-22af8d4c5198.png)

**Reply by mrdsaunders** [reply]
That is wonderful, thanks! Sorry for missing it.

Thank you also for the helpful other information.
